NuRadioReco/modules/io/coreas/readCoREASShower.py

```diff
@@ -103,20 +108,26 @@ def run(self):
 
             # add simulated pulses as sim station
             for idx, (name, observer) in enumerate(f_coreas['observers'].items()):
-                station_id = antenna_id(name, idx)  # returns proper station id if possible
+                # returns proper station id if possible
+                station_id = antenna_id(name, idx)
 
                 station = NuRadioReco.framework.station.Station(station_id)
                 if self.__det is None:
-                    sim_station = coreas.make_sim_station(station_id, corsika, observer, channel_ids=[0, 1, 2])
+                    sim_station = coreas.make_sim_station(
+                        station_id, corsika, observer, channel_ids=[0, 1, 2])
                 else:
-                    sim_station = coreas.make_sim_station(station_id, corsika, observer, channel_ids=self.__det.get_channel_ids(self.__det.get_default_station_id()))
+                    sim_station = coreas.make_sim_station(
+                        station_id, corsika, observer, channel_ids=self.__det.get_channel_ids(self.__det.get_default_station_id()))
                 station.set_sim_station(sim_station)
                 evt.set_station(station)
                 if self.__det is not None:
                     position = observer.attrs['position']
                     antenna_position = np.zeros(3)
-                    antenna_position[0], antenna_position[1], antenna_position[2] = -position[1] * units.cm, position[0] * units.cm, position[2] * units.cm
-                    antenna_position = cs.transform_from_magnetic_to_geographic(antenna_position)
+                    antenna_position[0], antenna_position[1], antenna_position[2] = - \
+                        position[1] * units.cm, position[0] * \
+                        units.cm, position[2] * units.cm
```

Comment:
[No way this is PEP8](https://peps.python.org/pep-0008/#maximum-line-length)! I think the PEP8 way would be to either turn this into a tuple
```
... = (
    -position[1] * units.cm,
    position[0] * units.cm,
    position[2] * units.cm
)
```
or, alternatively, just turn this into three separate lines. `\` is ugly!

Reply:
better?

Reply:
Much <3

Reply:
I wonder why my formatter did not complain. I agree that was not good style at all.